docker-for-aws/persistent-data-volumes.md

```diff
@@ -0,0 +1,45 @@
+---
+description: Persistent Data Volumes
+keywords: aws persistent data volumes
+title: Docker for AWS Persistent Data Volumes
+---
+
+## What is Cloudstor?
+Cloudstor is a Docker managed volume plugin that comes pre-installed (and pre-configured) in swarms deployed on Docker for AWS. Using volumes created through Cloudstor, swarm tasks can mount a persistent data volume that stays attached to the swarm tasks no matter which swarm node they get scheduled or migrated to. Currently Cloudstor heavily relies on shared storage infrastructure provided by the cloud platform to allow swarm tasks to create/mount their persistent volumes on any node in the swarm. In a future release we will introduce support for direct attached storage to satisfy very low latency/high IOPs requirements.
+
+## Using Cloudstor
+After creating a swarm on Docker for AWS and SSHing into any of the managers, you should be able to find cloudstor already installed and configured for the stack/resource group:
+
+```
+$ docker plugin ls
+ID                  NAME                        DESCRIPTION                       ENABLED
+f416c95c0dcc        docker4x/cloudstor:aws-v1.13.1-beta18   cloud storage plugin for Docker   true
+```
+**Note**: Make note of the plugin tag name, it will change between versions, and yours may be different then listed here.
+
+You can create swarm services that requires data persistence using the `--mount` flag and specifying Cloudstor as the driver. Here are some examples:
```

Comment:
Maybe "The following examples show how to create swarm services that require data persistence using the `--mount` flag and specifying Cloudstor as the driver."